client/components/commititems.jsx

```diff
@@ -1,13 +1,12 @@
 import React, {Component} from 'react';
-import CommitItem from './commititem';
+import CommitItem from './commitItem';
```

Comment:
the filename is commititem locally, all lowercase. We'll refactor filenames as a group later.